Fix the below breakage while compiling using glibc 2.25 MB toolchain
 | tc_core.c: In function 'tc_calc_size_table':
 | tc_core.c:190:29: error: 'UINT16_MAX' undeclared (first use in this function)
 |    if ((sz >> s->size_log) > UINT16_MAX) {
 |
 |                            ^~~~~~~~~~

Signed-off-by: Manjukumar Matha <manjukumar.harthikote-matha@xilinx.com>
Upstream-Status: Pending
----
--- iproute2-4.7.0-orig/tc/tc_core.c	2016-08-08 08:58:39.000000000 -0700
+++ iproute2-4.7.0/tc/tc_core.c	2017-07-28 11:16:52.110860333 -0700
@@ -12,6 +12,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include <unistd.h>
 #include <syslog.h>
 #include <fcntl.h>
